webrender_traits/src/channel_ipc.rs

```diff
@@ -19,9 +21,16 @@ pub type PayloadSender = IpcBytesSender;
 
 pub type PayloadReceiver = IpcBytesReceiver;
 
-impl PayloadHelperMethods for PayloadSender {
-    fn send_vec(&self, data: Vec<u8>) -> Result<(), Error> {
-        self.send(&data)
+impl PayloadSenderHelperMethods for PayloadSender {
+    fn send(&self, data: Payload) -> Result<(), Error> {
+        self.send(&data.to_data())
```

Comment:
is there any benefit of heap-allocating the vector, serializing the data to it, and then sending, versus just sending the data right away during serialization?